src/PanoViewer/PanoViewer.js

```diff
@@ -442,7 +455,8 @@ export default class PanoViewer extends Component {
 		this._photoSphereRenderer
 			.bindTexture()
 			.then(() => this._activate())
-			.catch(() => {
+			.catch(e => {
+				console.error(e);
```

Comment:
@WoodNeck 

Did you print the log before?

Reply:
Nope, that's my mistake. Thanks for checking